.github/workflows/ux-pages.yml

```diff
@@ -0,0 +1,55 @@
+name: Deploy UX to GitHub Pages
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["features/ux"]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
Does this need anything or is it intentionally blank? If so, maybe leave a comment so people know it's intentional.